src/server/project.ts

```diff
@@ -1748,7 +1747,16 @@ namespace ts.server {
                     useCaseSensitiveFileNames: this.program.useCaseSensitiveFileNames(),
                 };
             }
-            return this.projectService.host;
+            return {

```

Comment:
I put this here to provide a ModuleResolutionHost which uses the projectService's [fs]host for fileExists, directoryExists, realpath, useCaseSensitiveFileNames instead of `this.program.fileExists/directoryExists/realpath/useCaseSensitiveFileNames`. Maybe it's not needed? I don't know.